spec/controllers/api/v1/stops_controller_spec.rb

```diff
@@ -119,18 +119,18 @@
   end
 
   describe 'GET show' do
-    it 'returns stops by OnestopID' do
+    it 'returns stops by Onestop ID' do
       get :show, id: @metro_embarcadero.onestop_id
-      expect_json_types({
+      expect_json_types('stop',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.